app/javascript/chat/connectReducer.js

```diff
@@ -0,0 +1,320 @@
+import * as Type from './components/ChatTypes';
+export function connectReducer(state, action) {
+  const { type, payload = {} } = action;
```

Comment:
refactor (blocking): There is no need to give payload a default value of `{}`. This will actually mask potential errors when a payload isn't being sent that may be required by an action.